packages/core/src/Stores/segment-store.js

```diff
@@ -0,0 +1,63 @@
+import {
+    action,
+    computed }                  from 'mobx';
+import BinarySocket             from '_common/base/socket_base';
+import { isLoginPages }         from '_common/base/login';
+import { get as getLanguage }   from '_common/language';
+import BaseStore                from './base-store';
+import { getAppId }             from '../config';
+
+export default class SegmentStore extends BaseStore {
+    // only available on production
+    is_applicable = /^(16929|19111)$/.test(getAppId());
+    language = getLanguage().toLowerCase();
+
+    /**
+     * Contains binary user id
+     *
+     * @returns {object}
+     */
+    @computed
+    get userId() {
```

Comment:
Why have a a computed for getUserId on the SegmentStore when it's from the client store?

Reply:
youre right, we can get directly from client store, but, there are so many id's in client store. my intention was to make the specific id used to identify client in segment store. does computed value affect performance?